code/game/objects/items/his_grace.dm

```diff
@@ -177,6 +180,10 @@
 	var/victims = 0
 	meal.visible_message("<span class='warning'>[src] swings open and devours [meal]!</span>", "<span class='his_grace big bold'>[src] consumes you!</span>")
 	meal.adjustBruteLoss(200)
+	if(meal.mind.has_antag_datum(/datum/antagonist/changeling)) // If His Grace eats a changeling, it can't revive statis while inside
```

Comment:
But why

Reply:
Played a game last night where I was able to revive out of deadchat as a ling captured by his grace
It felt buggy
If it's not a bug that I can do this (???) I can remove it